addons/units/functions/fnc_onAltLoad.sqf

```diff
@@ -0,0 +1,40 @@
+#include "script_component.hpp"
+
+params ["_ctrl"];
+
+private _alt = (ctrlParent _ctrl) ctrlCreate [QGVAR(alt), -1];
+
+private _units = (vehicle ace_player) call FUNC(altUnits);
+private _altInfo = [_units select 0] call FUNC(altInfo);
```

Comment:
`[_units select 0]` is the same as `_units`, because FUNC(altInfo) only has one arg and as internal function with this also being unlikely to change, this line should be simplified.